src/io/flutter/editor/PreviewsForEditor.java

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
This class isn't enabled in the actual running application as the inline widget previews are disabled without an experimental flag to enable them as described earlier.